students/matiasli/Lesson03/mailroom.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# a_namesoace = {'Name', 'Donations'}
+
+a_namespace = [ ['Bob', [40, 5, 25]], ['Alice', [100, 100, 100]], ['Zin', [30, 15, 50]], ['Chloe', [40, 13, 19]], ['James', [15]] ]
+
+
+# prompt the user (you) to choose from a menu of 3 actions: “Send a Thank You”, “Create a Report” or “quit”)
+response = 0
+name_found = False
+
+while response != '3':
+    response = input("Select number: \n 1 - Send thank you \n 2 - Create report \n 3 - quit> ")
+    if response == '1':
+        response2 = input("What is the name to thank? or type 'list' to see all the names >")
+        if response2 == 'list':
+            # show full list
+            for name in a_namespace:
+                print(name[0])
+        else:
+            for name in a_namespace:
+                if response2 == name[0]:
+                    # we have a matdh, get donation amount, thank them
+                    response3 = input("What is amount they donated? >")
+                    #print("{}".format(response3))
+                    #print("{}".format(name[1]))
+                    name[1].insert(len(name[1]), int(response3))
+                    #print("{}".format(name[1]))
+
+                    name_found = True
+                    break
+                else:
+                    # add user to list
+                    name_found = False
+                    a_namespace.insert(len(a_namespace), (response2,[]))
+
+            print("email of thanks to {} for your donation".format(response2))
+
+    if response == '2':
+        # Include Donor Name, total donated, number of donations and average donation amount as values in each row
+        print(len(a_namespace))
+        for name2 in a_namespace:
```

Comment:
this should have a column header and then data only in the columns, just like the example in the assignment

this is also missing the sorting feature here